docs/examples/text/cjkText.js

```diff
@@ -0,0 +1,86 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Box, Flex, Text } from 'gestalt';
+
+export default function cjkText(): Node {
+  function NarrowShell({ text, useKeepAll }: {| text: string, useKeepAll?: boolean |}) {
+    return (
+      <Box width={24} color="dark">
+        <Text overflow="normal" size="200" color="inverse">
+          {useKeepAll ? <div style={{ wordBreak: 'keep-all' }}>{text}</div> : text}
+        </Text>
+      </Box>
+    );
+  }
+
+  const languages = [
+    {
+      id: 'English',
+      code: 'en',
+      lang: 'Your account options and more',
+    },
+    {
+      id: 'Chinese',
+      code: 'zh',
+      lang: '帐户和更多选项',
+    },
+    {
+      id: 'Korean',
+      code: 'ko',
+      lang: '계정 및추 가옵션',
+    },
+  ];
+
+  return (
+    <Box
+      marginTop={10}
+      padding={4}
+      width="100%"
+      height="100%"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      direction="column"
+    >
+      <Flex direction="column" gap={8}>
+        <Box>
+          <Box marginBottom={2}>
```

Comment:
marginBottom={2} might not be needed with Flex'sgap

Reply:
removed with above change